src/shared/constants.ts

```diff
@@ -21,4 +21,12 @@ export const constants = {
   queueName: "queueName",
   xAzureSettings: "x-azure-settings",
   entryPoint: "entryPoint",
-}
\ No newline at end of file
+  variableKeys: {
```

Comment:
why `variableKeys`?

Reply:
Keys for the `serverless.variables` object. We had just been using strings